pkg/controller/core.oam.dev/v1alpha2/application/apply.go

```diff
@@ -51,33 +47,13 @@ import (
 	oamutil "github.com/oam-dev/kubevela/pkg/oam/util"
 )
 
-func errorCondition(tpy string, err error) runtimev1alpha1.Condition {
-	return runtimev1alpha1.Condition{
-		Type:               runtimev1alpha1.ConditionType(tpy),
-		Status:             corev1.ConditionFalse,
-		LastTransitionTime: metav1.NewTime(time.Now()),
-		Reason:             runtimev1alpha1.ReasonReconcileError,
-		Message:            err.Error(),
-	}
-}
-
-func readyCondition(tpy string) runtimev1alpha1.Condition {
-	return runtimev1alpha1.Condition{
-		Type:               runtimev1alpha1.ConditionType(tpy),
-		Status:             corev1.ConditionTrue,
-		Reason:             runtimev1alpha1.ReasonAvailable,
-		LastTransitionTime: metav1.NewTime(time.Now()),
-	}
-}
-
```

Comment:
why these deleted?

Reply:
it's not deleted, it's moved  to `application_controller.go` where it's used.